sdk/search/azure-search-documents/src/main/java/com/azure/search/documents/models/SimpleField.java

```diff
@@ -0,0 +1,76 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.search.documents.models;
+
+/**
+ * A helper Field model to build a simple search field.
+ */
+public class SimpleField {
+    private String name;
+    private PrimitiveType dataType;
+    private Boolean facetable;
+    private AnalyzerName analyzer;
+    private AnalyzerName searchAnalyzer;
+    private AnalyzerName indexAnalyzer;
+
+    public String getName() {
+        return name;
+    }
+
+    public SimpleField setName(String name) {
+        this.name = name;
+        return this;
+    }
+
+    public PrimitiveType getDataType() {
+        return dataType;
+    }
+
+    public SimpleField setDataType(PrimitiveType dataType) {
+        this.dataType = dataType;
+        return this;
+    }
+
+
+    public Boolean getFacetable() {
+        return facetable;
+    }
+
+    public SimpleField setFacetable(Boolean facetable) {
+        this.facetable = facetable;
+        return this;
+    }
+
+    public AnalyzerName getAnalyzer() {
```

Comment:
This an others below are not supported on `SimpleField`. Only `SearchableField`.